lldb/source/Plugins/Language/Swift/SwiftOptional.cpp

```diff
@@ -172,8 +183,12 @@ bool lldb_private::formatters::swift::SwiftOptionalSummaryProvider::
   if (!target_valobj)
     return false;
 
-  PointerOrSP some = ExtractSomeIfAny(target_valobj, true);
+  std::optional<ValueObjectSP> some_or_err =
+      ExtractSomeIfAny(target_valobj, true);
+  if (!some_or_err)
+    return false;
 
+  ValueObjectSP some = *some_or_err;
   if (!some)
     return true;
```

Comment:
this is existing code, but should this be `false`?

Reply:
If we're here then we know the value is `.none`. Which I think it makes sense to return true for.